tests/CVTracker_Tests.cpp

```diff
@@ -0,0 +1,125 @@
+/**
+ * @file
+ * @brief Unit tests for openshot::Frame
+ * @author Jonathan Thomas <jonathan@openshot.org>
+ * @author FeRD (Frank Dana) <ferdnyc@gmail.com>
+ *
+ * @ref License
+ */
+
+/* LICENSE
+ *
+ * Copyright (c) 2008-2019 OpenShot Studios, LLC
+ * <http://www.openshotstudios.com/>. This file is part of
+ * OpenShot Library (libopenshot), an open-source project dedicated to
+ * delivering high quality video editing and animation solutions to the
+ * world. For more information visit <http://www.openshot.org/>.
+ *
+ * OpenShot Library (libopenshot) is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General Public License
+ * as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * OpenShot Library (libopenshot) is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with OpenShot Library. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "UnitTest++.h"
+// Prevent name clashes with juce::UnitTest
+#define DONT_SET_USING_JUCE_NAMESPACE 1
+#include "OpenShot.h"
+//#include "ProcessingController.h"
+#include <QImage>
+
+using namespace openshot;
+
+SUITE(CVTracker_Tests)
+{
+
+    // Just for the tracker constructor, it won't be used
+    ProcessingController processingController;
+
+    TEST(Track_Video)
+    {
+        // Create a video clip
+        std::stringstream path;
+        path << TEST_MEDIA_PATH << "test.avi";
+
+        // Open clip
+        openshot::Clip c1(path.str());
+        c1.Open();
+
+        // Create tracker
+        CVTracker kcfTracker("{\"protobuf_data_path\": \"\", \"tracker_type\": \"KCF\", \"bbox\": {\"x\": 294, \"y\": 102, \"w\": 180, \"h\": 166}, \"first_frame\": 0}", processingController);
```

Comment:
As a general suggestion, [raw string literals](https://en.cppreference.com/w/cpp/language/string_literal) (the ugly-but-handy <code>R"<var>delim</var>( ... )<var>delim</var>"</code> format strings) are your friend in these situations. See the JSON tests I just added to a few classes in #586, like:

https://github.com/OpenShot/libopenshot/blob/db738cecea98ae47e24703670d9cc15ede9a33c2/tests/Color_Tests.cpp#L165-L176